### DISCUSSION TITLE
Tour location

### DISCUSSION THREAD

**Discussion by Thompson-ongithub** [mytourbook/mytourbook#1273] [General]
How does MTB decide, which of the 2 locations it keeps and which get's deleted when combining duplicated locations with the One action in the Locations view context menu?
Because for me it always seems to keep the wrong one and to delete the once I actually want to keep (since it contains more information). It almost seems to do so on purpose, just to show me who's the boss. ;)
Is there a way to choose the one to be kept?

**Comment by wolfgang-ch** [comment]
> Is there a way to choose the one to be kept?

I knew that this question will come up sometimes 😄 (as you can see it in the second screenshot, the answer is highlighted)

The answer is a bit "hidden"

When you read the `One` tooltip it shows which actions are performed...

![image](https://github.com/mytourbook/mytourbook/assets/1283445/93940c5d-b4ef-4f92-97c5-75407e8890f8)

...when you read the `Combine...` tooltip then your question should be answered.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/ebc9d675-6fdf-4fe4-bd0b-f00a5bd3d870)

The trick here is, to sort the locations (by any column) that the the first location is the one you wanted, sometimes you have to resort it a second time for the locations which could not be included in the first sort. I did this workflow many times and it worked

**Reply by Thompson-ongithub** [reply]
So I was just unlucky that every single time it was the second location I wanted to keep ;)
**Thanks for the quick help.** 
Simply changing the sort order helped and now I can keep those locations I want
